setup.py

```diff
@@ -105,7 +105,7 @@ def run(self):
     packages=find_packages("src"),
     zip_safe=False,
     install_requires=install_requires,
-    extras_require={"dev": dev_requires},
+    extras_require={"dev": dev_requires, "rabbitmq": ["amqp==2.6.0"]},
```

Comment:
Could you add a note here to bump after a new release is cut that includes https://github.com/celery/py-amqp/pull/328?

Or, pinning to 2.6.x would be better. But not sure if they'll release as a patch or minor.

Reply:
Resolved by https://github.com/getsentry/sentry/pull/20112.